terraform/environments/digital-prison-reporting/modules/domains/ingestion-jobs/glue.tf

```diff
@@ -74,3 +74,88 @@ module "glue_reporting_hub_batch_job" {
   )
 }
 
+# Glue Job, Check All Raw Files Have Been Processed Job
```

Comment:
This Glue job checks that there is no pending unprocessed raw file by checking they have all been committed.